resources/logback.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<configuration>
+<configuration scan="true" scanPeriod="15 seconds">
```

Comment:
Is this a good value for production? Does it need to be done so often?

Reply:
It's not used just in production - it's also used when starting the app with `lein run`. Thus it needs to be low enough to not slow down the developer. 5 seconds would probably be better; 15 seconds is quite long time to wait when you're debugging something.

The scan only checks the `java.io.File#lastModified` of the configuration files (see `ch.qos.logback.core.joran.spi.ConfigurationWatchList#changeDetected`), so it's a cheap operation.